src/main/java/cn/nukkit/network/protocol/AnimateEntityPacket.java

```diff
@@ -0,0 +1,50 @@
+package cn.nukkit.network.protocol;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author IWareQ
+ */
+public class AnimateEntityPacket extends DataPacket {
+
+    public static final byte NETWORK_ID = ProtocolInfo.ANIMATE_ENTITY_PACKET;
+
+    public String animation;
+    public String nextState;
+    public String stopExpression;
+    public String controller;
+    public float blendOutTime;
+    public List<Long> entityRuntimeIds = new ArrayList<>();
```

Comment:
Please, to avoid compatibility problems when Cloudburst Nukkit adds a class with this same name, make these fields private and add getters and setters for them? Also, add annotations for the getters and setters. The private fields don't need them, for example:
```java
private String animation;

@PowerNukkitOnly
@Since("FUTURE")
public String getAnimation(){
    return animation;
}

@PowerNukkitOnly
@Since("FUTURE")
public void setAnimation(animation){
    this.animation = animation;
}
```